libflux/include/influxdata/flux.h

```diff
@@ -56,6 +61,10 @@ struct flux_error_t *flux_ast_marshal_json(struct flux_ast_pkg_t *, struct flux_
 // using flux_free if it is non-null.
 struct flux_error_t *flux_ast_marshal_fb(struct flux_ast_pkg_t *, struct flux_buffer_t *);
 
+// flux_get_env_stdlib instantiates a flatbuffers TypeEnvironment and creates a pointer
+// to it to use when performing lookups on the stdlib
+void flux_get_env_stdlib(struct flux_buffer_t *);
```

Comment:
Why did these `flux.h` files need to change? I am also confused as to why they seem to have changed in different ways. 

Was this because the `generate` step has been getting ignored?

Reply:
Yes, they got out of sync, because we were ignoring `checkgenerate`. The file `libflux/go/libflux/flux.h` had moved ahead of `libflux/include/...`.  

I also moved `flux_get_env_stdlib` to be earlier in the file so that the declaration of `flux_semantic_pkg_t` was just before `flux_analyze()`, since they are closely related.

Reply:
Great makes sense. Thanks!